main/src/ca/uwaterloo/flix/language/phase/unification/BoolFormula.scala

```diff
@@ -117,6 +125,7 @@ object BoolFormula {
     case Not(f1) => Not(substitute(f1, m))
     case And(f1, f2) => And(substitute(f1, m), substitute(f2, m))
     case Or(f1, f2) => Or(substitute(f1, m), substitute(f2, m))
+    case Xor(f1, f2) => Xor(substitute(f1, m), substitute(f2, m))
```

Comment:
I would really like to do Xor in a separate PR. Can we break it out into that?

Reply:
Yes that is no problem. It was not a very big change anyway